skins/laika/templates/Base_Layout.html.twig

```diff
@@ -18,6 +18,10 @@
     <meta property="og:site_name" content="Gemeinnützige Wikimedia Fördergesellschaft mbH" />
     <meta property="og:description" content="Mir liegt Wikipedia am Herzen, deshalb habe ich gespendet. Denn was wäre das Netz ohne Wikipedia?" />
 
+{#    TRIGGER WARNING - I REALLY NEED THIS UNTIL WE HAVE A GOOD SOLUTION OR ACCEPT GOOGLE AS YOUR MASTER -----> TODO #}
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+    {#    TRIGGER WARNING - I REALLY NEED THIS UNTIL WE HAVE A GOOD SOLUTION OR ACCEPT GOOGLE AS YOUR MASTER -----> TODO #}
```

Comment:
I think we should just hardcode the build directory. This way we don't sell to google, it will always be there on production and locally we just have to build it once.